src/der.rs

```diff
@@ -171,8 +172,9 @@ pub fn small_nonnegative_integer(input: &mut untrusted::Reader)
 /// Parses a positive DER integer, returning the big-endian-encoded value, sans
 /// any leading zero byte.
 #[inline]
-pub fn positive_integer<'a>(input: &mut untrusted::Reader<'a>)
-                            -> Result<untrusted::Input<'a>, error::Unspecified> {
+pub fn positive_integer<'a>
+    (input: &mut untrusted::Reader<'a>)
```

Comment:
Yes, it seems like maybe a rustfmt bug? When it does this to a `where` clause it puts the opening brace on a new line. Not sure why it doesn't do that in this case.